aztk/spark/models/plugins/jupyter_lab/jupyter_lab.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This custom script only works on images where jupyter is pre-installed on the Docker image
+#
+# This custom script has been tested to work on the following docker images:
+#  - aztk/python:spark2.2.0-python3.6.2-base
+#  - aztk/python:spark2.2.0-python3.6.2-gpu
+#  - aztk/python:spark2.1.0-python3.6.2-base
+#  - aztk/python:spark2.1.0-python3.6.2-gpu
+
+if  [ "$IS_MASTER" = "1" ]; then
+    conda install -c conda-force jupyterlab
+
+    PYSPARK_DRIVER_PYTHON="/.pyenv/versions/${USER_PYTHON_VERSION}/bin/jupyter"
+    JUPYTER_KERNELS="/.pyenv/versions/${USER_PYTHON_VERSION}/share/jupyter/kernels"
+
+    # disable password/token on jupyter notebook
+    jupyter lab --generate-config --allow-root
+    JUPYTER_CONFIG='/.jupyter/jupyter_notebook_config.py'
+    echo >> $JUPYTER_CONFIG
+    echo -e 'c.NotebookApp.token=""' >> $JUPYTER_CONFIG
+    echo -e 'c.NotebookApp.password=""' >> $JUPYTER_CONFIG
+
+    # get master ip
+    MASTER_IP=$(hostname -i)
+
+    # remove existing kernels
+    rm -rf $JUPYTER_KERNELS/*
+
+    # set up jupyter to use pyspark
+    mkdir $JUPYTER_KERNELS/pyspark
+    touch $JUPYTER_KERNELS/pyspark/kernel.json
+    cat << EOF > $JUPYTER_KERNELS/pyspark/kernel.json
+{
+    "display_name": "PySpark",
+    "language": "python",
+    "argv": [
+        "python",
+        "-m",
+        "ipykernel",
+        "-f",
+        "{connection_file}"
+    ],
+    "env": {
+        "SPARK_HOME": "$SPARK_HOME",
+        "PYSPARK_PYTHON": "python",
+        "PYSPARK_SUBMIT_ARGS": "--master spark://$MASTER_IP:7077 pyspark-shell"
+    }
+}
+EOF
+
+    # start jupyter notebook from /mnt - this is where we recommend you put your azure files mount point as well
```

Comment:
does jupyterlab have the same restriction as jupyter where you can't navigate up the directory structure? Right now `/mnt` is cluttered with a lot of things, it would be nice to have jupyterlab's default directory point somewhere else.

Reply:
Same limitation - you cannot navigate 'up' from /mnt. Since this is where we place samples it seems like a reasonable place to put it - where else do you think it could work?

Reply:
No that's fine, `/mnt/` is the best place since that is where their azure files will be mounted,